Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Reflection.PortableExecutable;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static float globalHeight = 0f;
```

Comment:
avoid using global, please move this to local scope. Global can be changed from anywhere and in any more complex cases than just this class you will hit yourself to the wall just because you made something global.

Reply:
I recently went through and re-worked the solution to remove global variables.

Reply:
Did you push? It seems to be the same